scheduler.go

```diff
@@ -283,11 +294,16 @@ func (s *Scheduler) StartAt(t time.Time) *Scheduler {
 // StartImmediately sets the Jobs next run as soon as the scheduler starts
 func (s *Scheduler) StartImmediately() *Scheduler {
 	job := s.getCurrentJob()
-	job.nextRun = s.time.Now().In(s.loc)
+	job.nextRun = s.time.NowRoundedDownToSeconds(s.loc)
 	job.startsImmediately = true
 	return s
 }
 
+// shouldRun returns true if the Job should be run now
+func (s *Scheduler) shouldRun(j *Job) bool {
```

Comment:
nice, much better

Reply:
yeah! had to change it because the job had no location concept, so it was setting it's next run in a different timezone which was messing with things